class.jetpack-debugger.php

```diff
@@ -230,6 +230,9 @@ public static function jetpack_debug_display_handler() {
 
 		$tests = array();
 
+		$tests['XML']['result']       = ( function_exists( 'xml_parser_create' ) ) ? 'PASS' : false;
+		$tests['XML']['fail_message'] = esc_html__( 'Jetpack can not load necessary XML manipulation libraries. This can happen if XML support in PHP is not enabled on your server. XML support is required for WordPress and Jetpack, please enable it or contact your hosting provider.', 'jetpack' );
```

Comment:
I wonder if instead of this simple message, we should instead add a link to a support doc on Jetpack.com, with more information like you've included in your PR description. This way users could send that support doc link to their host so the host can quickly understand exactly what is needed.

Reply:
That would work for me. I'll make that change real quick.

Reply:
93108b958eebc10bbb5a21cfff196ca7a194241d